### PR TITLE
Business logic & API connect for document upload in app release process

### PR DESCRIPTION
**Pull request by lavanya-bmw**

### CODE REVIEW

File: cx-portal/src/components/pages/AppReleaseProcess/components/AppMarketCard/index.tsx [hunk 1]
```diff
@@ -58,7 +59,7 @@ type FormDataType = {
   appLanguage: string[]
   price: string
   uploadImage: {
-    leadPictureUri: string
+    leadPictureUri: any
```
Comment (nidhigarg-bmw) [inline cx-portal/src/components/pages/AppReleaseProcess/components/AppMarketCard/index.tsx:62]: Need to give particular datatype here

File: cx-portal/src/components/pages/AppReleaseProcess/components/AppPage/index.tsx [hunk 2]
```diff
@@ -31,24 +31,27 @@ import HelpOutlineIcon from '@mui/icons-material/HelpOutline'
...
   longDescriptionDE: string
   images: string
-  uploadDataPrerequisits: string
-  uploadTechnicalGuide: string
-  uploadDataContract: string
-  uploadAppContract: string
+  uploadDataPrerequisits: any
+  uploadTechnicalGuide: any
+  uploadDataContract: any
+  uploadAppContract: any
```
Comment (nidhigarg-bmw) [inline cx-portal/src/components/pages/AppReleaseProcess/components/AppPage/index.tsx:54]: Why all these values with any datatype ?

Comment (lavanya-bmw) [reply cx-portal/src/components/pages/AppReleaseProcess/components/AppPage/index.tsx]: Fixed now

**Review by nidhigarg-bmw — APPROVED** [review]
Okay now

**Pull request merged**